packages/react-native-sdk/src/theme/colors.ts

```diff
@@ -6,44 +6,51 @@ const opacityToHex = (opacity: number) => {
     .padStart(2, '0');
 };
 
-const colors = {
+const lightColors = {
   primary: palette.blue500,
   error: palette.red400,
   info: palette.green500,
-  static_black: palette.grey950,
-  static_white: palette.grey50,
-  static_overlay: palette.grey950 + opacityToHex(0.85),
-  static_grey: palette.grey700,
   disabled: palette.grey600,
+
+  background: palette.grey800,
+  border: palette.grey300,
+  bar: palette.grey50,
   text_low_emphasis: palette.grey500,
   text_high_emphasis: palette.grey950,
-  controls_bg: palette.grey50,
-  borders: palette.grey300,
-  overlay: palette.grey950 + opacityToHex(0.4),
-  overlay_dark: palette.grey950 + opacityToHex(0.6),
-  bars: palette.grey50,
-  content_bg: palette.grey950 + opacityToHex(0.05),
-  dark_gray: palette.grey800,
+
+  overlay_light: palette.grey950 + opacityToHex(0.4),
+  overlay_medium: palette.grey950 + opacityToHex(0.6),
+  overlay_heavy: palette.grey950 + opacityToHex(0.85),
+
+  static_white: palette.grey50,
+  static_grey: palette.grey700,
+  static_black: palette.grey950,
 };
 
-const darkThemeColors = {
+const darkColors = {
   primary: palette.blue500,
   error: palette.red400,
   info: palette.green500,
-  static_black: palette.grey950,
-  static_white: palette.grey50,
-  static_overlay: palette.grey950 + opacityToHex(0.85),
-  static_grey: palette.grey700,
   disabled: palette.grey600,
+
+  background: palette.grey800,
+  border: palette.grey950 + opacityToHex(0.05),
+  bar: palette.grey900,
   text_low_emphasis: palette.grey500,
   text_high_emphasis: palette.grey50,
-  controls_bg: palette.grey900,
-  borders: palette.grey700,
-  overlay: palette.grey950 + opacityToHex(0.4),
-  overlay_dark: palette.grey50 + opacityToHex(0.6),
-  bars: palette.grey900,
-  content_bg: palette.grey950 + opacityToHex(0.05),
-  dark_gray: palette.grey800,
+
+  overlay_light: palette.grey950 + opacityToHex(0.4),
+  overlay_medium: palette.grey950 + opacityToHex(0.6),
+  overlay_heavy: palette.grey950 + opacityToHex(0.85),
+
+  static_white: palette.grey50,
+  static_grey: palette.grey700,
+  static_black: palette.grey950,
```

Comment:
All these should turn to generic names in my view, we should never be using a name called as `static_black` it should be something like `accent` or `primary` etc

Reply:
this is a good point. we don't use color names in the theme variables for React:
- https://github.com/GetStream/stream-video-js/blob/main/packages/styling/src/_global-theme-variables.scss

Reply:
Yes, you are right, we should not mention color names in the theme so I am renaming them like this:
```ts
  // old static_(white, gray, black)
  base1: palette.grey50,
  base2: palette.grey300,
  base3: palette.grey700,
  base4: palette.grey950,

  // old overlay_* or similar less frequently and wrongly used bar, border etc colors
  background1: palette.grey950 + opacityToHex(0.05),
  background2: palette.grey800,
  background3: palette.grey950 + opacityToHex(0.4), 
  background4: palette.grey950 + opacityToHex(0.6), 
  background5: palette.grey900,
  background6: palette.grey950 + opacityToHex(0.85),
```